sonic-xcvrd/scripts/xcvrd

```diff
@@ -573,7 +587,7 @@ class DaemonXcvrd(DaemonBase):
     def __init__(self):
         DaemonBase.__init__(self)
 
-        self.timeout = 1
+        self.timeout = XCVRD_MAIN_THREAD_SLEEP_SECS
         self.stop = threading.Event()
```

Comment:
Can you rename this "self.stop_event" so that it is clear it is an event, not a bool or something else?

Reply:
@jleveque changed all 'stop' to 'stop_event'.